cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillPreparer.java

```diff
@@ -652,9 +653,16 @@ private V1ObjectMeta createJob(V1ObjectMeta metadata, Map<String, RuntimeSpecifi
         .endTemplate()
       .endSpec()
       .build();
-
-    batchV1Api.createNamespacedJob(kubeNamespace, job, "true", null, null);
-    LOG.trace("Created Job {} in Kubernetes.", metadata.getName());
+    try {
+      batchV1Api.createNamespacedJob(kubeNamespace, job, "true", null, null);
+      LOG.trace("Created Job {} in Kubernetes.", metadata.getName());
+    } catch (ApiException e) {
+      if (e.getCode() == HttpURLConnection.HTTP_CONFLICT) {
+        LOG.warn("The job already exists : {}.\n Ignoring resubmission of the job." , e.getResponseBody());
```

Comment:
shall we say kubernetes job? 
Also the pattern in logback is based on logback.xml. Does `\n` take effect as a new line character?

Reply:
makes sense. revised the message.